src/Logging/InitializationLogger.cs

```diff
@@ -29,7 +29,7 @@ private static ILoggingBuilder LoadInitializationLogger(this ILoggingBuilder bui
 				builder.AddConsole();
 			}
 
-			builder.AddOmexLogging();
+			builder.AddOmexLogging(null!); // Because this method cannot access HostBuilder's configuration.
```

Comment:
Given that this change is breaking legacy logging for service initialization, it doesn't meet its own goal.